### PR TITLE
Changes type of "file" field in "Content" model

### PR DESCRIPTION
**Pull request by OrBin**

### CODE REVIEW

File: models/Content.js [hunk 1]
```diff
@@ -1,9 +1,15 @@
...
+    name: {
+      type: String,
+      required: true,
+    },
+    buffer: {
+      type: mongoose.Schema.Types.Buffer,
+      required: true,
+    },
   },
   title: {
```
Comment (bsuro10) [inline models/Content.js:18]: What is this title?
Isn't the file's name enough?

In the previous model we had several files (an array), here we just have content=file, is this what we want?

**Review by bsuro10 — CHANGES_REQUESTED** [review]

File: models/Content.js (hunk 1 above)
Comment (OrBin) [reply models/Content.js:18]: The title is a title for the content (for example, "Smartphones Lecture").
The file name is the actual file name (for example, "smartphones_lecture.pdf").

I think it's OK to have one file for each "content".

Comment (bsuro10) [reply models/Content.js:18]: I think we should give them the ability to upload multiple files.
If you want to split the lecture to 5 episodes that each one represent by a PDF file?

Comment (OrBin) [reply models/Content.js:18]: OK, I'll do that.

**Comment by OrBin** [comment]
@bsuro10 can you check if it's OK now?

**Review by bikov — APPROVED** [review]

**Pull request merged**